clang/include/clang/AST/CXXRecordDeclDefinitionBits.def

```diff
@@ -81,6 +81,9 @@ FIELD(IsStandardLayout, 1, NO_MERGE)
 ///   member.
 FIELD(IsCXX11StandardLayout, 1, NO_MERGE)
 
+/// True when the class has a virtual base class.
+FIELD(HasVBases, 1, NO_MERGE)
```

Comment:
Do we need to add this here? Doesn't `DefinitionData::NumVBases` suffice?

Reply:
Thanks for pointing this out!